testing/scala/APKBUILD

```diff
@@ -0,0 +1,111 @@
+# Contributor: norbjd
+# Maintainer:
+pkgname=scala
+pkgver=2.12.2
+pkgrel=0
+pkgdesc="Scala 2.12.2"
+url="http://scala-lang.org"
+arch="all"
+license="BSD3"
+
+# bash          : needed for scripts (e.g. scala)
+# openjdk8-jre  : needed for execution 
+depends="bash openjdk8-jre"
+
+# openjdk8      : needed for sbt + compilation
+# sbt           : needed for compilation
+# graphviz      : needed for diagrams generation in scaladoc
+makedepends="openjdk8 sbt graphviz"
+
+# documentation is separated
+subpackages="${pkgname}-doc"
+
+source="${pkgname}-${pkgver}.tar.gz::https://github.com/scala/scala/archive/v2.12.2.tar.gz"
```

Comment:
It might be more maintainable to make the URL use `$pkgname/archive/v$pkgver.tar.gz`.